Mathlib/Data/ENNReal/Basic.lean

```diff
@@ -502,7 +506,7 @@ def ofNNRealHom : ℝ≥0 →+* ℝ≥0∞ where
 #align ennreal.coe_of_nnreal_hom ENNReal.coe_ofNNRealHom
 
 -- TODO: generalize some of these (and subsequent lemmas about `smul`) to `WithTop α`
-section Actions
+section Actions -- This section can be moved into a different file
```

Comment:
How do you feel about moving this section to keep the `Basic` file small?

Reply:
Adressed on the other branch.